Stripe/STPCardScanner.swift

```diff
@@ -0,0 +1,450 @@
+//
+//  STPCardScanner.swift
+//  Stripe
+//
+//  Created by David Estes on 8/17/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+import AVFoundation
+import Foundation
+import UIKit
+import Vision
+
+enum STPCardScannerError: Int {
```

Comment:
public @objc

Reply:
The Error object that uses this is displayed by the STPAddCardViewController, it isn't exposed through any public API